tox.ini

```diff
@@ -54,6 +54,7 @@ deps =
     figure-!devdeps: mpl-animators==1.1.1
     figure-!devdeps: astropy==6.1.1
     figure-!devdeps: ndcube==2.2.2
+    build_docs-notebooks: git+https://github.com/sunpy/ndcube
```

Comment:
why?

Reply:
Because without recent additions to ndcube one of the plots in the visualisation notebook takes too long and times out, which breaks the build. I thought we'd fixed this for the workshop PR so I'm not sure why it's a problem again now. Obviously this isn't a good long-term solution.

Reply:
I don't think there's a fix to ndcube which is newer than the last bugfix release though? Let's not fold this in here, put it in it's own PR if we really need it?